pysb/core.py

```diff
@@ -1508,6 +1508,10 @@ def all_components(self):
             cset_all |= cset
         return cset_all
 
+    @property
+    def comps(self):
```

Comment:
The more I think about this, the more I think it should be spelled out in full as `components`. If someone is using it frequently enough that the name's length is cumbersome, then even `comps` would be too long. They would want to alias it with a local variable with as short a name as possible.

Reply:
Rationale for model.components makes sense. Implemented.